src/index.js

```diff
@@ -12,7 +12,7 @@ import { exec, execNode } from 'sb-exec'
 let NamedRegexp = null
 export const FindCache = new Map()
 
-export function rangeFromLineNumber(textEditor: TextEditor, line: number, column: number): Range {
+export function rangeFromLineNumber(textEditor: TextEditor, line: number, column: ?number): Range {
```

Comment:
Since this function can accept just a `textEditor` as an argument, should `line` be listed as optional as well?